CondTools/SiStrip/plugins/SiStripThresholdBuilder.cc

```diff
@@ -1,8 +1,31 @@
-#include "CondTools/SiStrip/plugins/SiStripThresholdBuilder.h"
 #include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+#include "CommonTools/ConditionDBWriter/interface/ConditionDBWriter.h"
+#include "CondFormats/SiStripObjects/interface/SiStripThreshold.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "CLHEP/Random/RandFlat.h"
+#include "CLHEP/Random/RandGauss.h"
+
 #include <iostream>
 #include <fstream>
 
+class SiStripThresholdBuilder : public edm::one::EDAnalyzer<> {
+public:
+  explicit SiStripThresholdBuilder(const edm::ParameterSet& iConfig);
+
+  ~SiStripThresholdBuilder() override = default;
+
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+
+private:
+  edm::FileInPath fp_;
+  uint32_t printdebug_;
```

Comment:
Same as above

Reply:
No, as you can see here it's used to limit the amount of printout.

https://github.com/cms-sw/cmssw/blob/8583b733fd4047cdd9fbeb285110bdae83c73c07/CondTools/SiStrip/plugins/SiStripThresholdBuilder.cc#L36

I am dismissing your comment.